src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -174,6 +175,11 @@ static RegistryBuilder<ForgeWorldPreset> getWorldTypesRegistryBuilder()
         return makeRegistry(WORLD_TYPES, ForgeWorldPreset.class).disableSaving().disableSync();
     }
 
+    static RegistryBuilder<BiomeModifierSerializer<?>> getBiomeModifierSerializersRegistryBuilder()
+    {
+        return makeRegistry(BIOME_MODIFIER_SERIALIZERS, c(BiomeModifierSerializer.class)).disableSaving().disableSync();
+    }
```

Comment:
This makes no sense to be inconsistent about. The other new registry builder is inlined; this one should be too.